### ISSUE TITLE
Accessibility declaration

### ISSUE THREAD

**Issue by elizapapaki** [DARIAH-ERIC/dariah-website#168]
We need to have an accessibility declaration when we launch the website. Who should be preparing that?

**Comment by stefanprobst** [comment]
does dariah have an acessibility declaration anywhere else already?

**Comment by elizapapaki** [comment]
No it will be the first one having one. We have a [legal notice](https://www.dariah.eu/legal-notice/) and [privacy notice](https://www.dariah.eu/privacy-notice/) but not an accessibility declaration.

**Comment by vronk** [comment]
@bwiechec-pcss do you think PCSS can provide such a declaration?
I assume you would have a lot of experience with that.

@stefanprobst can you chip in, what is the procedure here?
is it just text, or do some tests need to be run?

**Comment by stefanprobst** [comment]
first, we should clarify whether we think dariah.eu is legally required to include an accessibility statement.

if yes, it should look something like https://www.oeaw.ac.at/en/oeaw/accessibility

if no, a short paragraph stating that we aim for WCAG 2.1 AA should be enough.

**Comment by vronk** [comment]
@elizapapaki, can you check with Anne, how legal this needs to be?

**Comment by vronk** [comment]
if this is legal requirement this becomes a release-blocker!

**Comment by elizapapaki** [comment]
Okay Anne came back saying that though it's not 100% clear if we are legally obliged to have such declaration, it is strongly recommended for ERICs to have an accessibility statement. 

For an ERIC established in France, the key issue is not the ERIC status itself, but whether its website falls within the scope of the French implementation of the EU Web Accessibility Directive (Directive (EU) 2016/2102).

Under French law, France's accessibility framework (RGAA) requires covered organizations to publish a detailed accessibility statement ("déclaration d’accessibilité") which must describe:
- a statement of compliance status ("totally compliant", "partially compliant", or "non-compliant")

- inaccessible content

- feedback mechanism (contact) and complaint mechanisms.

So something similar to what ÖAW provides.
In this case who can provide such a draft?

**Comment by babslgam** [comment]
i found this. but i don't speak french. 
also it shoud be clarified if the declaration itself has to be in french in our case.

https://accessibilite.numerique.gouv.fr/ressources/modele-de-declaration/
https://accessibilite.numerique.gouv.fr/obligations/declaration-accessibilite/

**Comment by elizapapaki** [comment]
Thank you for this example Barbara! I clarified internally that the accessibility declaration should be in English. How do we proceed?

**Comment by elizapapaki** [comment]
@bwiechec-pcss @pawelmarpawelmar can you please provide a draft for an accessibility statement for the new website? I suppose you have experience on that? Something like the ACDH example (https://www.oeaw.ac.at/en/oeaw/accessibility) or the examples shared by Barbara above. Let us know how we can help on that!

**Comment by bwiechec-pcss** [comment]
As I see I forgot to replay previously, I've talked to Ola and we may provide a template of Accessibiltiy Declaration, but we as PCSS won't take responsibility for it's final content.

**Comment by elizapapaki** [comment]
If you provide a template and draft the technical parts (front-end related things and their accessibility parameter) then I can have a look and finalise what's missing. But the declaration seems to be a largely technical document, hence my question/request to you.

**Comment by elizapapaki** [comment]
@bwiechec-pcss can you please provide a template on this and draft the technical parts that are relevant for the declaration?

**Comment by bwiechec-pcss** [comment]
We are currently working on page audit in order to create technical part of accessibility declaration and to fix some issues with WCAG. It should be audited until friday/monday and I'm working on fixing those issues and probably most of them will be fixed untill the end of next week.

**Comment by stefanprobst** [comment]
@bwiechec-pcss what's the status of this?

**Comment by elizapapaki** [comment]
We have an accessibility statement draft to release the website: 

[accessibility-statement_2026-07-10 (2).html.pdf](https://github.com/user-attachments/files/30895410/accessibility-statement_2026-07-10.2.html.pdf)

**Comment by babslgam** [comment]
we can close this once the responsible person (neither @stefanprobst nor @babslgam) signs off a final version.
